e2e_tests/test_performance.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+import os
+import hashlib
+import json
+import gzip
+import requests
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from .assets.serve import CORE_TIME
+import time
+
+
+
+current_path = os.path.abspath(os.path.dirname(__file__))
+checker_path = [f'{current_path}/assets/performance_event_trigger.py', f'{current_path}/assets/checker.py']
+modifier_path = [f'{current_path}/assets/flow_editor.py']
+REQUEST_NUM = 60
+AVU_DURATION = 2
+ALL_DURATION = 15
+
+
+def fetch_url(req_url):
+    response = requests.get(req_url)
+    return response.text
+
+
+def send_request(url):
+    with ThreadPoolExecutor() as executor:
+        futures = [executor.submit(fetch_url, url) for i in range(REQUEST_NUM)]
+        for future in as_completed(futures):
+            try:
+                res = future.result()
+                print(res)
+            except Exception as e:
+                print(f"Request Error:{e}")
+
+
+def test_performance(lyrebird, mock_server):
+    start_time = time.time()
+    send_request(lyrebird.uri_mock + mock_server.api_performance)
```

Comment:
lyrebird.uri_extra_mock

Reply:
uri_extra_mock have white list, test requests will be filtered out, so keep use uri_mock here.